.gitlab-ci.yml

```diff
@@ -0,0 +1,47 @@
+.bazel_docker_image: &bazel_docker_image thulioassis/bazel-docker-image:0.29.1
+.default_branch: &default_branch v3-develop
+
+stages:
+- lint
+- build
+- publish
+
+lint:
+  stage: lint
+  image: *bazel_docker_image
+  script:
+  - ./dev/linters/shellcheck.sh
+
+build:chart:
+  stage: build
+  image: *bazel_docker_image
+  script:
```

Comment:
I prefer to have scripts like these as a proper script file.
See `publish:chart:` for another place where I would prefer that.
See `lint` for a place where this is done properly (IMHO).

Reply:
You are right! I should've done this from the beginning. Thanks.

Reply:
Confirming it done, ty